### PR TITLE
ADEN-8214 MEGA Everywhere on mobile-wiki

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: app/modules/ads/ad-context.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 export default {
-  adUnitId: '/{custom.dfpId}/wka.{targeting.s0}/{custom.wikiIdentifier}//{targeting.s2}/{src}/{slotConfig.slotName}',
-  megaAdUnitId: '/{custom.dfpId}/{serverPrefix}1b.{slotConfig.group}/{slotConfig.adProduct}{slotConfig.slotNameSuffix}/'
+  adUnitId: '/{custom.dfpId}/{serverPrefix}1b.{slotConfig.group}/{slotConfig.adProduct}{slotConfig.slotNameSuffix}/'
```
Comment (fraszczakszymon) [inline app/modules/ads/ad-context.js]: use `{custom.serverPrefix}` please

File: app/modules/ads/setup.js [hunk 2]
```diff
@@ -83,22 +83,12 @@ function setupAdContext(adsContext, instantGlobals, isOptedIn = false) {
...
-  context.set('slots.incontent_player.adUnit', context.get('megaAdUnitId'));
-  context.set('slots.invisible_high_impact_2.adUnit', context.get('megaAdUnitId'));
-
-  context.set('slots.mobile_in_content.videoAdUnit', context.get('vast.megaAdUnitId'));
-  context.set('slots.incontent_boxad_1.videoAdUnit', context.get('vast.megaAdUnitId'));
-  context.set('slots.incontent_player.videoAdUnit', context.get('vast.megaDBNameAdUnitId'));
-  context.set('slots.video.videoAdUnit', context.get('vast.megaAdUnitId'));
-  context.set('slots.featured.videoAdUnit', context.get('vast.megaDBNameAdUnitId'));
+  let adUnitId = context.get('adUnitId');
+  adUnitId = adUnitId.replace('{serverPrefix}', utils.isProperCountry(['AU', 'NZ']) ? 'vm' : 'wka');
```
Comment (fraszczakszymon) [inline app/modules/ads/setup.js]:
```suggestion
  context.set('custom.serverPrefix', utils.isProperCountry(['AU', 'NZ']) ? 'vm' : 'wka')
```

Comment (dmnsobczak) [reply app/modules/ads/setup.js]: Done.

File: app/modules/ads/ad-context.js (hunk 1 above)
Comment (dmnsobczak) [reply app/modules/ads/ad-context.js]: Yep, sure can do.

**Review by fraszczakszymon — APPROVED** [review]

**Review by bckatarzyna — APPROVED** [review]

**Pull request merged**